packages/instrumentation-elasticsearch/src/elasticsearch.ts

```diff
@@ -0,0 +1,158 @@
+import { diag, context, suppressInstrumentation } from '@opentelemetry/api';
+import type elasticsearch from '@elastic/elasticsearch';
+import { ElasticsearchInstrumentationConfig } from './types';
+import {
+    InstrumentationBase,
+    InstrumentationConfig,
+    InstrumentationModuleDefinition,
+    InstrumentationNodeModuleDefinition,
+    InstrumentationNodeModuleFile,
+} from '@opentelemetry/instrumentation';
+import { VERSION } from './version';
+import { DatabaseAttribute } from '@opentelemetry/semantic-conventions';
+import { startSpan, onError, onResponse, defaultDbStatementSerializer, normalizeArguments } from './utils';
+import { ELASTICSEARCH_API_FILES } from './helpers';
+
+type Config = InstrumentationConfig & ElasticsearchInstrumentationConfig;
```

Comment:
i actually changes this and used ElasticsearchInstrumentationConfig directly instead